src/test/java/fi/otavanopisto/kuntaapi/test/ManagementMocker.java

```diff
@@ -0,0 +1,174 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import fi.metatavu.management.client.model.Announcement;
+import fi.metatavu.management.client.model.Banner;
+import fi.metatavu.management.client.model.Menu;
+import fi.metatavu.management.client.model.Page;
+import fi.metatavu.management.client.model.Post;
+import fi.metatavu.management.client.model.Tile;
+
+public class ManagementMocker extends AbstractMocker {
+  
+  private static final String BANNERS = "/wp-json/wp/v2/banner";
+  private static final String MENUS = "/wp-json/kunta-api/menus";
+  private static final String PAGES = "/wp-json/wp/v2/pages";
+  private static final String POSTS = "/wp-json/wp/v2/posts";
+  private static final String TILES = "/wp-json/wp/v2/tile";
+  private static final String ANNOUNCEMENTS = "/wp-json/wp/v2/announcement";
+
+  private List<Banner> bannerList = new ArrayList<>();
+  private List<Menu> menuList = new ArrayList<>();
+  private List<Page> pageList = new ArrayList<>();
+  private List<Post> postList = new ArrayList<>();
+  private List<Tile> tileList = new ArrayList<>();
+  private List<Announcement> announcementList = new ArrayList<>();
+  
+  public ManagementMocker mockBanners(String... ids) {
+    for (String id : ids) {
+      Banner banner = readBannerFromJSONFile(String.format("management/banners/%s.json", id));
+      mockGetJSON(String.format("%s/%s", BANNERS, id), banner, null);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "%s/%s" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)